packages/affiliates/test/mocha-mainnet/dappmining.js

```diff
@@ -4,7 +4,7 @@ const highland = require("highland");
 
 const { DappMining } = require("../../libs/affiliates");
 const { getWeb3 } = require("@uma/common");
-const { getAbi } = require("@uma/core");
+const { getAbi } = require("@uma/core-1-2-0");
```

Comment:
why are you fixing on this version?

Reply:
It's a great question.
- The getAbi call below requests that version specifically, which isn't supported by contracts-node.
- I didn't want to make it the current version of core because these old index functions depend on truffle artifacts which we are going to be dropping soon (and as a result, dropping the core index functions, most likely).
- Since these tests have a pretty deep dependency on this version (there is generated data that explicitly depends on it, so new data would need to be generated), I figured it was easier to just depend on an old version rather than try to create new hardcoded data to update it.

Thoughts?

Reply:
that makes sence. bit a of pain that you need to do this but what an you do 🤷

Reply:
yeah :/.